### PR TITLE
Tag IndexedArrays v0.2.0

### PR DESCRIPTION
**Pull request by garrison**
Updated to work on julia 0.5.

### CODE REVIEW

**Comment by tkelman** [comment]
push the git tag so it shows up under https://github.com/garrison/IndexedArrays.jl/releases

**Comment by garrison** [comment]
done.

**Comment by tkelman** [comment]
note that `release` in `.travis.yml` will become 0.5 soon, so best to test against 0.4 and 0.5 explicitly

File: IndexedArrays/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline IndexedArrays/versions/0.2.0/requires]: good idea to specify a minimum version that implements the features you need

**Comment by tkelman** [comment]
cc @diegozea this caused a failure in MIToS.jl: http://pkg.julialang.org/logs/MIToS_0.4.log

**Pull request merged**